File: patterns/c-patterns/pattern63.c
// Pattern 63: ...

// 5 5 5 5 5
//  * * * *
//   3 3 3
//    * *
//     1

#include <stdio.h>

int main()
{
    int i, j;
    int n = 5;

    for (i = n; i >= 1; i--)
    {
        for (j = n; j >= 1; j--)
        {
            if (i >= j)
            {
                if (i % 2 == 0)
                    printf("* ");
                else
                    printf("%d ", i);
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern63.py
# Pattern 63: ...

# 5 5 5 5 5
#  * * * *
#   3 3 3
#    * *
#     1

n = 5

for i in range(n, 0, -1):
    for j in range(n, 0, -1):
        if i >= j:
            if i % 2 == 0:
                print("* ", end="")
            else:
                print(f"{i} ", end="")
        else:
            print(" ", end="")
    print()
